src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardSyncer.java

```diff
@@ -210,28 +201,24 @@ private static void assertAllParentShardsAreClosed(Set<String> inconsistentShard
     }
 
     /**
-     * Note: this has package level access for testing purposes. 
+     * Note: this has package level access for testing purposes.
      * Useful for asserting that we don't have an incomplete shard list following a reshard operation.
      * We verify that if the shard is present in the shard list, it is closed and its hash key range
-     *     is covered by its child shards.
-     * @param shards List of all Kinesis shards
-     * @param shardIdsOfClosedShards Id of the shard which is expected to be closed
-     * @return ShardIds of child shards (children of the expectedClosedShard)
-     * @throws KinesisClientLibIOException
+     * is covered by its child shards.
      */
-    static synchronized void assertClosedShardsAreCoveredOrAbsent(Map<String, Shard> shardIdToShardMap,
-            Map<String, Set<String>> shardIdToChildShardIdsMap,
-            Set<String> shardIdsOfClosedShards) throws KinesisClientLibIOException {        
+    synchronized void assertClosedShardsAreCoveredOrAbsent(Map<String, Shard> shardIdToShardMap,
+                                                           Map<String, Set<String>> shardIdToChildShardIdsMap,
```

Comment:
Could you use the formatter from: https://github.com/awslabs/amazon-kinesis-client/blob/master/formatter/formatter.xml